sw/ground_segment/cockpit/live.ml

```diff
@@ -1521,6 +1532,45 @@ let get_shapes = fun (geomap:G.widget)_sender vs ->
 let listen_shapes = fun (geomap:G.widget) ->
   safe_bind "SHAPE" (get_shapes geomap)
 
+let get_cam_status = fun (geomap:MapCanvas.widget) _sender vs ->
```

Comment:
The gimbal information (pan and tilt angle) are currently taken from the telemetry CAM message (if sent by the drone, assuming 0 by default). With this, the `server` agent is computing a target point. Isn't this enough with altitude above ground to compute the footprint ? I don't really like the idea of sending the coordinates of the corners of the footprint. I prefer either a target point or absolute angles.